packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -95,7 +95,26 @@ const mapAttachmentType = (attachmentType: AttachmentType): FileMetadataAttachme
   return 'unknown';
 };
 
-/* @conditional-compile-remove(file-sharing) @conditional-compile-remove(teams-inline-images) */
+const processTeamsImageContent = (message: ChatMessageWithStatus): string | undefined => {
+  if (sanitizedMessageContentType(message.type).includes('html') && message.content?.attachments) {
+    let teamsImageHtmlContent = '';
+    const attachments: ChatAttachment[] = message.content?.attachments;
+    attachments.forEach((attachment) => {
+      if (attachment.attachmentType === 'teamsImage') {
+        teamsImageHtmlContent += `\r\n<p><img alt="image" src="" itemscope="${attachment.contentType}" id="${attachment.id}"></p>`;
```

Comment:
Do you want to have some CSS somewhere to cap the width of these images?

Reply:
I tested with a 4000x2000 dimension image and it seems to be fine (I'm guessing that it uses previewUrl so AMS resizes it)?
<img width="662" alt="image" src="https://github.com/Azure/communication-ui-library/assets/77021369/e6032e34-3509-4fcb-9ec8-180e837d4c29">